docs/api-docs/channels/channels-pos-tutorial.md

```diff
@@ -0,0 +1,159 @@
+# Building Point-of-Sale Channels
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Creating a channel](#creating-a-channel)
+- [Specifying the platform](#specifying-the-platform)
+- [Configuring UI sections](#configuring-ui-sections)
+- [Rendering UI sections](#rendering-ui-sections)
+- [Resources](#resources)
+
+</div>
+
+This article documents how to use [Channels Toolkit](https://developer.bigcommerce.com/api-docs/channels/guide/channels-toolkit-reference) to install a POS channel into [Channel Manager](https://support.bigcommerce.com/s/article/Selling-Everywhere-with-Channel-Manager) during a [single-click app's](https://developer.bigcommerce.com/api-docs/apps/guide/types) installation. We'll assume you're an experienced app and POS developer familiar with channels on BigCommerce.
+
+### Prerequisites
+* [Familiarity with channels on BigCommerce](https://developer.bigcommerce.com/api-docs/channels/guide/overview)
+* [Experience building apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro)
+* Experience building POS integrations
+
+## Creating a channel
+
+After receiving the [POST response](https://developer.bigcommerce.com/api-docs/apps/guide/auth#receiving-the-post-response) at the end of the [single-click app OAuth flow](https://developer.bigcommerce.com/api-docs/apps/guide/auth#receiving-the-post-response), create a channel on the merchant's store. This allows merchants to import sales from the POS system and configure settings. To create a POS channel, set `type` to `pos` in the [create a channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel) request.
+
+```http
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/channels
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "name": "Sample POS",
+  "type": "pos",
+  "platform": "custom",
+  "external_id": "",
+  "status": "connected",
+  "is_listable_from_ui": true,
+  "is_visible": true,
+  "config_meta": {
+    "app": {
+      "id": 112233,
+      "sections": [
+        {
+          "title": "Overview",
+          "query_path": "overview"
+        },
+        {
+          "title": "Import",
+          "query_path": "import"
+        },
+        {
+          "title": "Settings",
+          "query_path": "settings"
+        },
+      ]
+    }
+  }
+}
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel#requestrunner)
+
+[View reference documentation for this request](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/createchannel).

```

Comment:
This link takes you to the Create a Channel resource, same as on line 23. (https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel)